pyo3-macros-backend/src/module.rs

```diff
@@ -344,7 +348,15 @@ pub fn pymodule_module_impl(
             )
         }
     }};
-    let initialization = module_initialization(&name, ctx, module_def, options.submodule.is_some());
+    let initialization = module_initialization(
+        &name,
+        ctx,
+        module_def,
+        options.submodule.is_some(),
+        options
+            .supports_free_threaded
+            .is_some_and(|op| op.value.value),
```

Comment:
hmm, MSRV rust doesn't have `is_some_and`, I guess I should use an `if let` instead?

Reply:
I shared a link to this PR on the recurse center zulip and @nebkor suggested `.map.unwrap_or` which I think does the same thing. My brain kind of turned off when I saw the error on the MSRV tests...